.devcontainer/devcontainer.json

```diff
@@ -6,12 +6,14 @@
 		"vscode": {
 			"extensions": [
 				"ms-vscode.cpptools",
+				"ms-vscode.cpptools-extension-pack",
```

Comment:
What do we use the `cpptools-extension-pack` for?

They always try to force you to install this, did it slip in or do we want it.

If we are not using it, then we should blacklist it in `extensions.json`.

Reply:
I keep getting prompted to install it when I use the debugging tools, it seems to add:

```json
    "extensionPack": [
        "ms-vscode.cpptools", // we already have this installed
        "ms-vscode.cpptools-themes",
        "ms-vscode.cmake-tools"
    ]
```

What do you think? Should we still blacklist it?

Reply:
Yeah, there is an `unwantedRecommendations` mechanism to do so in the `extensions.json` file.

```json
"unwantedRecommendations": [
    "ms-vscode.cmake-tools",
    "ms-vscode.cpptools-extension-pack",
    "ms-vscode.cpptools-themes"
]
```

Reply:
I'm keeping cmake-tools because it's useful for working with the cmakefiles within zephyr but the others I'll exclude